### PR TITLE
Create Resource Overview and Approach.md

### PR DESCRIPTION
**Pull request by jared-lambert**
First draft

### CODE REVIEW

File: docs/Guidance/Resource Overview and Approach.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
+# Resource Overview and Approach
+## Approach
```
Comment (eddie-knight) [inline docs/Guidance/Resource Overview and Approach.md:2]:
```suggestion
# Resource Overview and Approach

## Approach

```

**Comment by ojeb2** [comment]
looks good - I will expand the CDMC item in the near future

File: docs/Guidance/Resource Overview and Approach.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+### Threats
+Many threat libraries exist today, but few are as widely adopted in financial services as the MITRE ATT&CK Framework. We seek to align with existing open industry best practices, and therefore are recommending MITRE ATT&CK be used as our base threat library. Read more about this reasoning here.
+
+### Controls
+Many existing control catalogs exist today, and this is a space with a diverse adoption landscape. We desire to use a controls framework that has broad adoption, is global, and will provide a wealth of mappings to both threat libraries and cloud provider evidence. 
+
+NIST 800-53 is our top pick in this space and is our recommended control framework. <Deep dive link>
+
+CDMC Notes Here<CDMC Notes>
+
```
Comment (mlysaght2017) [inline docs/Guidance/Resource Overview and Approach.md:30]: Will add in CSA CCM here in time too

**Review by mlysaght2017 — COMMENTED** [review]
Looks good

**Comment by jared-lambert** [comment]
Updated folder structure to match "resources" folder, per offline discussion. Will move Oscal resources in future merge from training.

File: docs/resources/readme.md [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+
+NIST 800-53 is our top pick in this space and is our recommended control framework.
+
+### Evidence
+When it comes to evidence, our focus is on the format and retrieval strategies to be used. We desire formats that are commonly adopted by practitioners, and generally available from service providers. 
+The OSCAL format is a leading industry standard that meets these needs and is our current recommendation. 
+
+
+
+
```
Comment (eddie-knight) [inline docs/resources/readme.md:37]:
```suggestion
```

**Review by eddie-knight — APPROVED** [review]
LGTM— Thanks Jared!

**Review by eddie-knight — APPROVED** [review]

**Comment by eddie-knight** [comment]
@TheJuanAndOnly99 do you know why I'm not able to merge approved PRs today? Something feels different.

**Pull request merged**